README.adoc

```diff
@@ -269,9 +272,9 @@ include::{common-includes}/devmode-build.adoc[]
 
 When the server is running, select either approach to fetch your system properties:
 
-- Visit the http://localhost:9080/inventory/systems/localhost[http://localhost:9080/inventory/systems/localhost^] URL. The URL retrieves the system property information for `localhost` host name by invoking the http://localhost:9080/system/properties[http://localhost:9080/system/properties^] service.
+* Visit the http://localhost:9080/inventory/systems/localhost[http://localhost:9080/inventory/systems/localhost^] URL. The URL retrieves the system property information for `localhost` host name by invoking the http://localhost:9080/system/properties[http://localhost:9080/system/properties^] service.
```

Comment:
does this mean that when the user visits http://localhost:9080/inventory/systems/localhost, it automatically resolves to http://localhost:9080/system/properties[http://localhost:9080/system/properties ?

Reply:
I'm not sure about

http://localhost:9080/system/properties service

is the URL itself a service Or is the service hosted by the URL?, lik 
"invoking the service at the http://localhost:9080/system/properties URL

just looking for consistency in terms

Reply:
When the user visits the inventory service at http://localhost:9080/inventory/systems/localhost, the inventory service makes a request to the system service at  http://localhost:9080/system/properties and displays that information and displays that information 

Regarding your second point, I have updated the text to make it more clear.

Reply:
> I have the same question here re endpoints vs URLs. Why are they referred to as URLs here instead of endpoints? Sorry to keep on about this but I'm trying to figure out the best way to be consistent throughout a guide and across the guides collection

Was this the line you were referring to in this comment here?